README.md

```diff
@@ -1,8 +1,9 @@
-﻿# Welcome to the psake project
+Welcome to the psake project
+=============================
 
-[![Build status][appveyor-badge]][appveyor-build]
-[![Build Status][travis-badge]][travis-build]
-[![Join the chat at https://gitter.im/psake/psake][gitter-badge]][gitter]
+[![Build status](https://ci.appveyor.com/api/projects/status/e8b90u1q1ex5hx9m?svg=true)](https://ci.appveyor.com/project/psake/psake)

```

Comment:
We lost the changes I made on this one.  I think we could just copy/paste the top and bottom of the file I committed to this one and we'd have it.

Reply:
Or if it would be easier, you can go ahead and get this committed and I can resubmit another PR to put back my changes.

Reply:
My fault. I'll add it back.